src/types.ts

```diff
@@ -35,3 +36,19 @@ export type QueryClientConfig = {
     isDataEqual?: typeof isDataEqual;
   };
 };
+
+export type OwnItemsQuery = {
+  page?: number;
+  name?: string;
+  limit?: number;
+};
```

Comment:
Not used anymore?

Reply:
indeed used, I think u're talking about `all`?
@pyphilia